pkgs/by-name/sn/sniffnet/package.nix

```diff
@@ -75,15 +84,21 @@ rustPlatform.buildRustPackage rec {
       }
   '';
 
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+  versionCheckProgramArg = "--version";
+  doInstallCheck = true;
+
   meta = {
     description = "Cross-platform application to monitor your network traffic with ease";
     homepage = "https://github.com/gyulyvgc/sniffnet";
-    changelog = "https://github.com/gyulyvgc/sniffnet/blob/v${version}/CHANGELOG.md";
+    changelog = "https://github.com/gyulyvgc/sniffnet/blob/v${finalAttrs.version}/CHANGELOG.md";
     license = with lib.licenses; [
       mit # or
       asl20
     ];
     maintainers = with lib.maintainers; [ figsoda ];
```

Comment:
Could you also add the NGI team to the maintainers, please? Since it's an [NGI project](https://nlnet.nl/project/Sniffnet/).

Reply:
Sure, done!